src/components/Accordion.js

```diff
@@ -24,46 +24,95 @@ document.addEventListener(
+}
+
+/**
+ * Toggles class that enables proper focus
+ * This is a workaround for the IE and Edge Browsers that do not support :focus-within
+ * @param {*} focusEvent
+ */
+const toggleAccordionFocus = focusEvent => {
+  const { target, type: focusType } = focusEvent;
+  var accordionHeaderElms = target.getElementsByClassName(
+    "dg_accordion_buttontext-holder"
+  );
+
+  if (focusType === "focusin") {
+    accordionHeaderElms[0].classList.add("ms-focus-within");
+  }
+
+  if (focusType === "focusout") {
+    accordionHeaderElms[0].classList.remove("ms-focus-within");
+  }
+};
+
+const selectElementByClassName = (element, cssNameText) => {
   var sibling = element.nextElementSibling;
   while (sibling) {
-    if (sibling.matches(cssName)) return sibling;
-    sibling = sibling.nextElementSibling;
+    if (sibling.matches(cssNameText)) {
+      return sibling;
+    } else {
+      sibling = sibling.nextElementSibling;
+    }
   }
 };
 
-const menuAction = element => {
-  var mainDiv = element.closest(".dg_accordion");
-  var menuItems = mainDiv.getElementsByClassName("multi-collapse");
-  var totalCollapsed = mainDiv.getElementsByClassName("show");
-  var buttonElement = element.closest("button");
-  var childDiv = selectElementByClassName(buttonElement, ".multi-collapse");
-  var buttonAll = mainDiv.getElementsByClassName("dg_allitems");
-
-  var isMenuOpen = childDiv.className.includes(menuOpen);
+const menuAction = accordionHeaderElm => {
+  var mainDivElm = accordionHeaderElm.closest(".dg_accordion");
+  var menuItems = mainDivElm.getElementsByClassName("multi-collapse");
+  var totalCollapsedPanels = mainDivElm.getElementsByClassName("show");
+  var accordionButtonElm = accordionHeaderElm.closest("button");
+  var accordionContentElm = selectElementByClassName(
+    accordionButtonElm,
+    ".multi-collapse"
+  );
+  var accordionButtonAllElms = mainDivElm.getElementsByClassName("dg_allitems");
+  var isMenuOpen = accordionContentElm.className.includes(menuOpen);
 
   //If its open then we want to close it and vice versa
-  collpasePanelUpdate(!isMenuOpen, childDiv);
-
-  buttonAll
-    ? updateButtonStatus(buttonAll[0], menuItems.length, totalCollapsed.length)
+  collpasePanelUpdate(!isMenuOpen, accordionContentElm);
+
+  accordionButtonAllElms
+    ? updateButtonStatus(
+        accordionButtonAllElms[0],
+        menuItems.length,
+        totalCollapsedPanels.length
+      )
     : null;
 };
 
 const allMenuItemsAction = button => {
   const isMenuOpen =
     button.textContent.toLowerCase().trim() === "open all" ? true : false;
 
-  var body = button.closest(".dg_accordion");
-  var menuItems = body.getElementsByClassName("multi-collapse");
-  var totalCollapsed = body.getElementsByClassName("show");
+  var bodyElm = button.closest(".dg_accordion");
+  var menuItems = bodyElm.getElementsByClassName("multi-collapse");
+  var totalCollapsedPanels = bodyElm.getElementsByClassName("show");
```

Comment:
As we talked about yesterday this probably becomes `totalCollapsedPanelElms` based on our new naming conventions [guidelines](https://github.com/baltimorecounty/dotgov-components/wiki/Naming-Things) but let's push this through.